pipelines/match_injected_tract_catalog.yaml

```diff
@@ -0,0 +1,34 @@
+description: Match tract level injected catalog and input catalog.
```

Comment:
"Match tract-level injected_deepCoadd_catalog datasets to injected_objectTable_tract datasets."